mmdt/mmdt/settings.py

```diff
@@ -185,16 +185,31 @@
     },
     'handlers': {
         'rotate_file': {
-            'level': 'DEBUG',
             'class': 'logging.handlers.RotatingFileHandler',
-            'filename': os.path.join(log_dir, 'django.log'),
-            'maxBytes': 10 * 1024 * 1024,
+            'filename': os.path.join(log_dir, 'django_rotated.log'),
+            'mode': 'a',
+            'maxBytes': 5 * 1024 * 1024, # if exceeds 5MB, a new log file will be created
             'backupCount': 5,
             'formatter': 'verbose',
         },
+        'timed_rotate_file': {
```

Comment:
If we write both rotate_file and timed_rotate_file, it comes out two logs in the log folder with same info. We better decide whether should we use rotate or time rotate.